src/Wallet.js

```diff
@@ -19,12 +19,42 @@ class Wallet {
      * @param {number} [chainId=1]
      * @param {string} [rpcServer=false]
      */
-    constructor(chainId, rpcServer = false) {
+    constructor(...params) {
+        const { chainId, rpcServer, rpcTimeout } = this._initializeParams(params)        
         this.chainId = chainId ? utils.isNumber(chainId) : undefined;
         this.Account = new Account(this)
         this.Transaction = new Transaction(this);
-        this.Rpc = new RPC(this, rpcServer);
+        this.Rpc = new RPC(this, rpcServer, rpcTimeout);
         this.Utils = utils;
     }
+
+    _initializeParams(params) {
+        let chainId, rpcServer, rpcTimeout;
+
+        if (params.length === 2) {
+            chainId = params[0];
+            rpcServer = params[1];
+        }
+        
+        if (params.length === 1 && typeof params[0] === "object") {
+            chainId = params[0].chainId;
+            rpcServer = params[0].rpcServer;
+            rpcTimeout = params[0].rpcTimeout;
+        }
+        
+        if (params.length === 1 && typeof params[0] === "string") {
+            rpcServer = params[0];
+        }
```

Comment:
Lets add comments to these cases so they are easier to understand a glance eg:

L34: Backwards compatibility catch
L39: Object Based configuration
L45: Shorthand instancing w/ RPC only